web/themes/custom/pineapple/sass/pineapple.styles.scss

```diff
@@ -2,9 +2,10 @@
 
 // Import external libraries. Grunt-sass has been configured to have the
 // theme's npm node_modules and Bower libraries directory as an include path.
-@import "breakpoint-sass/stylesheets/breakpoint";
+@import "../node_modules/breakpoint-sass/stylesheets/breakpoint";
```

Comment:
This change is not required or necessary please see the comment above and line 12 of your `Gruntfile.js` file and revert the changed

Reply:
The change doesn't affect the functionality of the code and removes php storm errors.

Reply:
I have now reverted the change due to the reasons discussed. Thank you.